mod-linked-data/src/main/resources/citation/mod-linked-data/features/lccn-pattern-validation/samples/invalid-instance-request.json

```diff
@@ -5,7 +5,7 @@
         {
           "http://bibfra.me/vocab/marc/Title":{
             "http://bibfra.me/vocab/marc/mainTitle":[
-              "Invalid instance main title"
```

Comment:
This title is almost same as the title used in create-bib-api test. Hence, search results return count '2' instead of '1' in create-bib-api  tests and hence cause failure.